extensions/ext/GLSL_EXT_ray_query.txt

```diff
@@ -114,6 +114,9 @@ Overview
       rayQueryGetIntersectionObjectToWorldEXT -> OpRayQueryGetIntersectionObjectToWorldKHR instruction
       rayQueryGetIntersectionWorldToObjectEXT -> OpRayQueryGetIntersectionWorldToObjectKHR instruction
 
+    Note: When mapping to SPIR-V, ray query objects are always passed by pointer, and it is invalid to access the underlying object.
+    This differs from other arguments, notably including the acceleration structure used to initialize a ray query, which are passed by value. 
```

Comment:
```suggestion
    This differs from other arguments, notably including the acceleration structure handle used to initialize a ray query, which are passed by value. 
```
Clarify that this is just the acceleration structure handle, not the whole acceleration structure being passed by value.